ddtrace/contrib/botocore/patch.py

```diff
@@ -161,7 +161,7 @@ def inject_trace_to_eventbridge_detail(params, span):
 
 
 def get_kinesis_data_object(data):
-    # type: (str, Optional[bool]) -> Optional[Dict[str, Any]]
+    # type: (str) -> Optional[Dict[str, Any]]
     """
     :data: the data from a kinesis stream
     :try_b64: whether we should try to decode the string as base64
```

Comment:
```suggestion
```

should probably update this outdated line too

Reply:
done